### ISSUE TITLE
Latest Docker image for Gitea-armv7 on raspberry pi 4 failed to start

### ISSUE THREAD

**Issue by Agrigor** [Kunde21/raspberry-dev#9]
Hi there,
I am using your docker container "kunde21/gitea-arm:latest" for a long time and after pulling the latest version today, the container doesn't start anymore.

The logfile is full of these errors:

```
2021-09-05T08:07:41.445250199Z s6-svscan: warning: unable to iopause: Operation not permitted
2021-09-05T08:07:41.446833654Z s6-svscan: warning: executing into .s6-svscan/crash
2021-09-05T08:07:41.446893839Z s6-svscan: fatal: unable to exec .s6-svscan/crash: No such file or directory
```

Do you have any idea why and what to do to fix that?

Kind regards and thanks for your effort!

**Comment by dulfox** [comment]
I had the same issue with v1.15
I had to go back to v1.14.5
There are lots of BREAKING with this new branch :  [https://github.com/go-gitea/gitea/releases/tag/v1.15.0](https://github.com/go-gitea/gitea/releases/tag/v1.15.0)
Hope Kundie will be able to fix it.

Édit : it works with the just released 1.15.1 ! Thx !

**Comment by Kunde21** [comment]
We needed to revert alpine 3.13 to 3.12 due to an s6 bug.  I updated alpine to 3.14 for v1.15, but it looks like a fresh new s6 bug is in the new release.  yay software :laughing:

**Comment by Kunde21** [comment]
Build v1.15.0 and v1.15.1 have been re-published with alpine 3.12 base

**Comment by Agrigor** [comment]
It works, thanks a lot :)
